client/my-sites/backup/rewind-flow/granular-restore.tsx

```diff
@@ -0,0 +1,275 @@
+import { Button, Card, Gridicon } from '@automattic/components';
```

Comment:
This is based on [client/my-sites/backup/rewind-flow/restore.tsx](https://github.com/Automattic/wp-calypso/blob/trunk/client/my-sites/backup/rewind-flow/restore.tsx), just removed everything that we don't require here (like the restore types selection).